crmsh/bootstrap.py

```diff
@@ -1321,10 +1321,29 @@ def init_sbd():
     SBD can also run in diskless mode if no device
     is configured.
     """
-    if not _context.sbd_device and _context.diskless_sbd:
+    def get_dev_list(dev_list):
+        result_list = []
+        for dev in dev_list:
+            if ';' in dev:
+                result_list.extend(dev.strip(';').split(';'))
+            else:
+                result_list.append(dev)
+        return result_list
+
+    # non-interactive case
+    if _context.sbd_device:
+        _context.sbd_device = get_dev_list(_context.sbd_device)
+        if len(_context.sbd_device) > 3:
+            error("Maximum number of SBD device is 3")
+        for dev in _context.sbd_device:
+            if not is_block_device(dev):
+                error("SBD device %s doesn't seem to exist" % (dev))
```

Comment:
I would put the same error message used in the interactive code (or update the other if you prefect this message).
`    {} doesn't look like a block device".format(dev_item)`

Reply:
OK, changed